cmd/bucket-policy.go

```diff
@@ -137,6 +152,8 @@ func initBucketPolicies(objAPI ObjectLayer) error {
 	globalBucketPolicies = &bucketPolicies{
 		rwMutex:             &sync.RWMutex{},
 		bucketPolicyConfigs: policies,
+		objAPI:              objAPI,
+		alwaysRefresh:       objAPI.StorageInfo().Backend.Type == FS,
```

Comment:
Would it not be better to use `globalObjectAPI` instead of storing it inside the bucket policies struct?